README.md

```diff
@@ -70,23 +70,23 @@ Or [download a ZIP of the files](https://github.com/digininja/DVWA/archive/maste
 - [Installing DVWA on Windows using XAMPP](https://youtu.be/Yzksa_WjnY0)
 - [Installing Damn Vulnerable Web Application (DVWA) on Windows 10](https://www.youtube.com/watch?v=cak2lQvBRAo)
 
-### Windows + XAMPP
-
-The easiest way to install DVWA is to download and install [XAMPP](https://www.apachefriends.org/) if you do not already have a web server setup.
+### Docker
 
-XAMPP is a very easy to install Apache Distribution for Linux, Solaris, Windows and Mac OS X. The package includes the Apache web server, MySQL, PHP, Perl, a FTP server and phpMyAdmin.
+The easiest way to run DVWA is with containers.
+To get started:
 
-This [video](https://youtu.be/Yzksa_WjnY0) walks you through the installation process for Windows but it should be similar for other OSs.
+0. Install [Docker](https://www.docker.com/) or your favorite container engine.
+1. Clone or download this repository and extract (see [Download](#download)).
+2. Open a terminal of your choice and change its working directory to `DVWA`.
+3. `docker compose up -d`
```

Comment:
I just tried this on Ubuntu and got:

```
DVWA $ docker compose up -d
unknown shorthand flag: 'd' in -d
See 'docker --help'.
```

Reply:
Tried without the `-d`

```
DVWA $ docker compose up 
docker: 'compose' is not a docker command.
See 'docker --help'
```

Reply:
```
robin@flint DVWA $ docker version
Client:
 Version:           20.10.21
 API version:       1.41
 Go version:        go1.18.1
 Git commit:        20.10.21-0ubuntu1~22.04.2
 Built:             Thu Mar  2 18:26:04 2023
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.21
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.18.1
  Git commit:       20.10.21-0ubuntu1~22.04.2
  Built:            Wed Feb 15 15:26:57 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.12-0ubuntu1~22.04.1
  GitCommit:        
 runc:
  Version:          1.1.4-0ubuntu1~22.04.1
  GitCommit:        
 docker-init:
  Version:          0.19.0
  GitCommit:        
```

Reply:
I don't have an Ubuntu bare metal computer to test this. How did you install Docker? It feels like you did a `apt install docker`.
[Their instructions](https://docs.docker.com/engine/install/ubuntu/#install-docker-engine) includes installing `docker-compose`, as is [their convenience install script](https://docs.docker.com/engine/install/ubuntu/#install-using-the-convenience-script).
Please try again and let me know.

Reply:
I probably installed it from the native package. If this is a problem then it needs to be fully documented as I imagine some users will already have docker installed this way and will then have problems when things don't work.

On my desktop, docker works for what I need and I'm not going to change that to test this in case it breaks stuff that I need. Could you create a VM to try it out?

Reply:
I have added references to install Docker and Docker Compose.
I have also tested and have DVWA working on Ubuntu 22.04.2.